### ISSUE TITLE
🤔[question] Get Slots per User

### ISSUE THREAD

**Issue by Leon0402** [determined-ai/determined#10184]
### Describe your question

Hi, 
is it possible to know for each user how many slots they use and ideally which tasks use these slots? Does determined cli support that?
There are different commands like `det user`, `det task`, `det shell`, but it is not obvious how to get the combined information from this.

### Checklist

- [X] Did you search [the docs](https://docs.determined.ai/latest/) for a solution?
- [X] Did you search github issues to find if somebody asked this question before?

**Comment by ioga** [comment]
Hello,

Today you'd probably have to do five commands `det [experiment|notebook|tensorboard|shell|command] list` and aggregate their output. they should all have `--csv` option which should make this easier to do programmatically.

**Comment by Leon0402** [comment]
@ioga Could you maybe give some more details? 

I think there might be a misunderstanding here. None of these commands give me the slots used. E.g. when I do for instance `det shell --all` I get:
```
id   | username   | description   | state      | exitStatus   |resourcePool   | workspaceName
```
With some preprocessing I can filter for active shells here. But I don't see a way to get the slots.

Maybe to give a motivation why this is needed: Multiple people use the cluster and can create shells (and other stuff) with reserved slots (gpus). As we have a maximum number of gpus, it would be really nice to get an overview over how many gpus are used by each user. Currently we do this manually in an excel, but people forget to update that.

**Comment by Leon0402** [comment]
Would be nice as a feature request to include that information directly in the table. Both for the cli and the graphical userinterface. At least that is an information that would be beneficial to us.

**Comment by ioga** [comment]
yep I agree, and I've created an internal ticket to track this request.
